crypto/src/e2e_identity/types.rs

```diff
@@ -0,0 +1,212 @@
+//! We only expose byte arrays through the FFI so we do all the conversions here
+
+use super::error::{E2eIdentityError, E2eIdentityResult};
+
+/// TODO
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+pub struct E2eiAcmeDirectory {
```

Comment:
same here as with the TS/WASM conversions, you might need a `#[serde(rename_all = "camelCase")]` directive